docs/samples/Microsoft.ML.Samples/Dynamic/ConvertType.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ConvertTypeTransform
```

Comment:
>ConvertTypeTransform [](start = 24, length = 20)

Please rename this to ConvertType (identical to the API extension method that it correspond to)